public/javascripts/SVLabel/src/SVLabel/canvas/ContextMenu.js

```diff
@@ -472,15 +472,38 @@ function ContextMenu (uiContextMenu) {
             _setTags(targetLabel);
             _setTagColor(targetLabel);
             if (getStatus('disableTagging')) { disableTagging(); }
+
+            // Hide the severity menu for the Pedestrian Signal label type.
+            if (labelType === 'Signal') {
+                $severityMenu.css({visibility: 'hidden', height: '0px'});
+            } else {
+                $severityMenu.css({visibility: 'inherit', height: '50px'});
+            }
             var windowHeight = $menuWindow.outerHeight();
+            var connectorBuffer = 3; // for connector to overlap border of label icon
+
+            var connectorHeight = parseInt(window.getComputedStyle($connector[0]).getPropertyValue("height"));
+            var connectorWidth = parseInt(window.getComputedStyle($connector[0]).getPropertyValue("width"));
 
             // Determine coordinates for context menu when displayed below the label.
-            var topCoordinate = labelCoord.y + 20;
-            var connectorCoordinate = -5;
+            var topCoordinate = labelCoord.y
+                + svl.LABEL_ICON_RADIUS
+                + parseInt(connectorHeight)
+                - connectorBuffer;
+            var connectorCoordinate = -1 * parseInt(connectorHeight);
 
             // Determine coordinates for context menu when displayed above the label.
```

Comment:
Maybe this comment should be rewritten to:
>If there isn't enough room to show the context menu below the label, determine coords to display above.

So that it's clear why we're doing this.